run_tests.sh

```diff
@@ -34,8 +36,10 @@ tree --version &&
 ping -V &&
 # docker should be installed
 docker --version &&
-# composer should be installed
-composer --version &&
```

Comment:
composer without providing an env should work as well
same output as composer 2.0